Code/Tools/TestImpactFramework/Runtime/Native/Code/Source/TestRunner/Native/Job/TestImpactNativeTestRunJobData.h

```diff
@@ -24,6 +24,13 @@ namespace TestImpact
         {
         }
 
+        //! Copy and move constructors/assignment operators.
+        NativeTestRunJobData(const NativeTestRunJobData& other);
+        NativeTestRunJobData(NativeTestRunJobData&& other);
+        NativeTestRunJobData& operator=(const NativeTestRunJobData& other);
+        NativeTestRunJobData& operator=(NativeTestRunJobData&& other);
```

Comment:
These functions can be = defaulted.
All the implementations do is the same thing the default copy/move constructor and assignment operators would do.
```suggestion
        //! Copy and move constructors/assignment operators.
        NativeTestRunJobData(const NativeTestRunJobData& other) = default;
        NativeTestRunJobData(NativeTestRunJobData&& other) = default;
        NativeTestRunJobData& operator=(const NativeTestRunJobData& other) = default;
        NativeTestRunJobData& operator=(NativeTestRunJobData&& other) = default;
```

Reply:
Fixed 👍